crates/api_models/src/payments.rs

```diff
@@ -1380,6 +1385,14 @@ pub struct BankTransferNextStepsData {
     pub receiver: ReceiverDetails,
 }
 
+#[derive(Clone, Debug, Eq, PartialEq, serde::Serialize, serde::Deserialize, ToSchema)]
+pub struct VoucherNextStepData {
+    // Reference number required for the transaction
+    reference: String,
+    // Url to download the payment instruction
+    download_url: Option<Url>,
+}
```

Comment:
Can we have these as documentation comments itself (```/// ...```)?

Reply:
Done👍